zoo-css-challenge/style.css

```
@@ -108,28 +117,65 @@ footer {
   border: 5px solid #747c92;
 }
 
+/*2) Bears */
+/* images vertically aligned to the top of the container*/
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  vertical-align: top;
+  border: 5px solid #617fd3;
+}
+
+.image-container img:nth-of-type(2) {
+  vertical-align: top;
+  border: 5px solid #6b0579;
 }
 
+.image-container img:nth-of-type(3) {
+  vertical-align: top;
+  border: 5px solid #e2500c;
+}
+
+
+/* End 2*/
+
+
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
 
+/* 3) Tigers */
+.tigers h2, h3{
```

```suggestion
.tigers h2, .tigers h3{
```
The `h3` selector will change all the `h3` elements on the page, but we only want to change the `h3` in the tigers section so we can update it to `.tigers h3` instead